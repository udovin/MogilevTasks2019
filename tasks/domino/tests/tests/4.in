454
-54352396 81289
-31067642 5746
-41100133 7028
-47389636 10424
-51217971 19094
-55601229 20657
-32160048 44289
-55706496 34160
-35372502 16211
-52982682 9717
-56974993 38128
-34913112 11671
-29207232 29657
-55139783 9787
-46760517 17169
-51485945 25126
-38354555 82756966
-54476009 18259
-31255005 17278
-48051685 132487
-46128789 30934
-34646248 23921259
-61619230 49174837
-47882520 2062
-50791541 30198
-59619597 15233
-56179303 10937
-46172939 101975
-58529704 105138
-41878273 36626
-61296661 89640
-30438725 53566
-49047131 16891
-45749745 27465
-47402647 137820
-63184288 59575
-48430362 4217640
-33658018 73851616
-60149662 124991
-29035999 34665
-52667264 1022
-32774336 72799
-59436956 75935
-41985392 10118
-53960103 23865483
-54271894 55316
-42530767 19987
-39357371 74293
-51948650 72175
-52320514 74676
-46380947 12909
-45665997 30258
-59836164 64378
-43748403 63735
-48048301 3192
-39058959 25182
-40503014 92064
-44790407 2907
-30744907 90866
-29956204 50970
-52809554 26359
-51349181 40451
-39482538 16927
-60981353 9524
-32407001 73407
-53455099 84576
-43412032 111614
-36658853 1681
-34406907 121687
-47790412 19639875
-29593181 78142
-59663601 1890
-50457520 43576
-41419632 44693
-40339701 23311
-62148299 10494
-42933932 3797
-42983338 37719
-55809706 13175
-36984042 26129
-55334849 29879
-33103146 94869
-46854015 41061
-38437995 17628
-58100442 24064
-34783823 27284
-58063766 23712
-45410482 92276
-57808194 35639
-37578630 30693679
-43269417 27828
-62437229 30641
-39585254 38206
-34193922 1077
-35742303 35944
-54542746 32880
-35060692 11642
-41374673 3188
-29272310 47603
-44034282 4991
-55784830 488
-57709422 53271
-58930264 999
-45639852 4268
-31346868 58059
-41522824 21832
-31085346 4444
-56208839 2841
-38107780 68665
-32558677 57330
-56479112 7401
-55694421 4303
-56245507 17957
-49907769 13247
-61499818 59562
-33361507 91618
-35300153 54592
-37875416 98664931
-36519221 44054
-54901107 23731
-63401939 53172
-39298162 1759
-29814907 1240
-53105648 21698
-30293901 11629
-62544662 45555
-42553453 29635
-31845693 19032
-54736695 22745
-39964887 70984
-51881518 44966
-32651147 100212
-52839234 28785
-62257384 15332
-53615045 74168
-62961685 16776
-32547332 7249
-34034877 19813
-42361822 19535
-34358524 10017
-57399841 42376
-48909077 11560
-44445780 36064
-34278053 36808
-60416665 6505
-63163798 40
-55035080 12434
-58446970 54419
-43053210 56718
-31720692 7286
-43951014 21684
-60697094 84253
-50721850 35182
-47168707 15132
-54690262 46038
-53405399 34145
-40102239 2146
-59550974 35808
-60008309 12656
-35541034 86276
-51606987 43883
-55741163 14421
-29859278 93051
-41736143 37668
-49820921 83259
-37983609 64919
-60287521 9510
-54228940 31095
-35494169 158
-33896293 75114
-50517357 33217
-48330385 10311
-43347801 38101
-52091578 19673
-52202886 853
-53692662 89340
-63115359 23571
-53884836 14796
-37430196 78380
-56578944 98595
-62669942 1361
-45544574 65237
-35223986 50858
-35841069 1224
-41115512 10966
-47664015 123127
-39849517 13285
-38710044 54112
-48705018 2503
-48778731 119591
-35167538 38053
-37325037 66771387
-50183107 39763
-58138675 2975
-29104400 25354
-33520583 72138
-32496122 6767
-51085281 67324
-61757386 25087
-60905395 63455
-38843882 8258041
-46908698 132571
-42261345 92165
-57883521 26101
-53360954 5038
-31281863 16
-38666108 34700
-44900144 13284
-62796744 77745
-50843976 45128
-41268084 56644
-56535608 2694
-38545569 22058
-56125155 45703
-54097212 6282
-45171853 9058
-52154442 43013
-62425788 5591
-47279440 39496
-48416627 3184
-34581340 2283
-40043253 51099
-49442570 70076919
-55481727 63027
-58342303 3036264
-49958982 88341
-41054442 12541
-34377807 9752
-55855278 39199803
-41608120 70419
-63264116 12245
-62099492 1666
-58026018 22577
-33754227 110122
-31281887 9392
-30333674 3829
-34833415 14616
-32086245 2442
-55391700 7596
-39433451 14781
-63014019 1755
-56855818 12556
-36215797 34807
-56023691 38400
-36319271 52431
-48570241 63270
-57299228 2364
-42152800 30242
-44914628 85291
-57095442 13427
-39136080 92461
-40802712 43894
-36870484 113236
-38951460 95171187
-55557543 1594
-53542617 54022
-43632646 93582
-54208017 17834
-44303437 74699
-40107690 49991
-31757049 76594
-36394251 73651
-29701670 31341
-45994902 98952
-56288383 8022
-52401049 2129
-49707296 42560
-53906587 37455
-49556316 510
-34977153 23098
-58924978 3730
-33033495 1832
-55118258 8488
-35120307 41333
-30141271 5285
-42384083 7133
-48674227 2721
-50319837 85668
-47046842 79014
-44199331 39597
-37514022 62910
-35941775 109703
-32003180 47795
-49069473 4874
-32266274 51529047
-34536896 30763
-47222462 32260
-58832397 26114
-32889627 9718
-49162294 22389
-62322140 85725
-47543987 80267
-40322043 456
-62891752 41051
-53177985 30404
-48344000 45928
-54759915 22701
-41380398 22649
-53949188 4650
-39118908 13751
-40168065 147
-31622835 49396
-55947897 46289
-44141424 10445
-54869103 28274
-55212676 7074
-40938361 17303
-40258789 11487
-37781830 24960
-42040017 99743
-51418661 20922
-56376944 16961
-31491894 11045
-33454909 44537
-58674288 21041
-53821550 6265
-31876428 52527
-36756140 61318
-55175987 30
-56023084 68
-46593457 4781
-30683824 13352
-50232632 73696
-49079795 2162
-41209182 2947
-36061378 47213
-54110105 15097956
-35081278 35107
-29481871 79474
-60560595 27872
-40605218 4968
-50973341 19417
-45130995 24823
-61134311 35903
-59718373 2035
-34133229 765
-49568032 42772
-44019392 4449
-42470268 37965
-46476905 38270
-47974247 66603
-37694431 80837
-49374467 20276
-39720393 100371
-49399200 933
-45022892 91503
-60372970 32363
-37101519 81782
-45984668 9084
-42033310 6047
-35393067 11708
-40170358 58245461
-53375042 13861
-57467909 3070
-39232197 55879
-30880248 25291
-36662133 90737
-44687954 57563
-49289628 66605
-38337179 2043
-40279312 10531
-61201915 71810
-46754212 4792
-29010929 20579
-51821510 32744
-62052739 17596
-48932545 81793
-58701448 47914
-29675503 20205
-30171293 14553
-54970850 37577
-45243039 46875
-33232663 118473
-56715694 104419
-42856351 30453
-30666005 12347
-57636586 61188
-58165594 17710
-45861710 110332
-58223261 31235
-59076984 7028
-51261465 34929
-31117411 133271
-61058656 66020
-43895003 48705
-41240210 19302
-41471558 5822
-38191917 38499572
-30725774 6292
-48258245 22257
-60784461 11470
-48192584 55938
-34064429 26020
-44131587 5923
-44003645 1562
-59614329 4558
-54455483 15968
-54650770 16206
-46294284 31102
-57150979 16811
-61797614 85466
-34884394 7327
-36112556 49239
-50600338 85957
-52243836 59902
-40709179 84416
-51701630 35321
-59314298 33609
-30078462 59767
-52675110 34065
-42738710 85698
-29370268 76331
-46625324 80440
-58445948 618
-55176104 14488
-43175903 26070
-43544897 21586
-40931102 4040
-42622833 25532
-45366866 38378
-59904769 101735
-59184990 50072746
-30959091 104442
-39898064 16570
-53221489 31224
-41682085 52662
-40449245 41765
-55563160 11674
-49784106 36450
-61932764 43943
-30052677 2504
-57178601 97822
-60524115 36129
-60472516 20635
-58183695 20235
-57545247 83997
-42587300 2719
-37196127 32659
-34724638 48418
-34934392 32792
-48716741 943
-45313574 7326
-63093221 9261
-35860431 61820
-35646285 87025
-60059322 63154
-62117915 23700
-44566388 108466
-41039176 4178
-30553745 41686
-59668290 15632
-52526792 125762
-50058991 15217
-61440378 43167
