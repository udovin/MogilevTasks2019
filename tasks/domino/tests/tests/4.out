4 1 1 1 1 1 1 2 1 1 1 1 1 1 2 1 118 1 1 3 1 71 431 1 1 2 1 2 3 1 1 1 1 1 3 2 252 58 3 2 1 1 1 1 323 3 1 2 2 1 1 2 1 1 1 1 2 1 4 1 1 1 1 1 1 3 2 1 3 242 1 1 1 3 2 1 1 1 1 1 1 2 1 1 1 1 1 2 1 110 1 2 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 2 1 1 1 1 2 1 1 113 1 1 1 1 1 1 1 1 2 1 1 2 1 3 1 1 2 1 1 1 1 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 2 2 1 1 1 2 1 1 2 1 2 1 1 2 1 1 1 1 1 1 1 3 2 1 1 2 1 1 1 1 1 2 1 2 1 4 1 107 1 1 1 2 1 1 1 1 123 2 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 1 1 1 1 1 267 1 391 2 1 1 357 1 1 1 1 59 2 1 1 1 1 1 1 1 1 2 1 1 1 1 2 1 3 1 1 124 1 1 2 1 1 2 2 1 2 2 1 1 1 2 1 1 1 1 1 3 1 1 1 1 1 1 1 3 1 1 44 1 1 1 1 1 2 1 1 1 1 2 1 2 1 1 1 1 1 2 1 1 2 1 1 3 1 1 1 1 1 1 2 1 1 1 1 1 2 1 1 1 325 2 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 1 1 1 1 1 1 1 1 142 1 1 1 1 2 1 1 1 1 2 1 1 1 1 2 2 1 1 1 1 1 1 1 1 1 1 1 1 1 1 3 1 1 1 1 116 1 1 1 1 1 1 1 1 1 1 1 1 2 1 1 2 2 1 1 1 2 2 1 1 2 1 2 1 1 1 1 1 2 401 2 1 1 1 1 2 1 1 1 3 1 1 2 2 1 1 1 2 1 1 1 2 1 2 2 1 1 1 2 2 1 1 
